include/swift/Frontend/FrontendOptions.h

```diff
@@ -258,6 +258,13 @@ class FrontendOptions {
   /// By default, we include ImplicitObjCHeaderPath directly.
   llvm::Optional<std::string> BridgingHeaderDirForPrint;
 
+  /// Disable implicitly built Swift modules because they are explicitly
+  /// built and given to the compiler invocation.
+  bool DisableImplicitModules = false;
+
+  /// The paths to a set of explicitly built modules from interfaces.
+  std::vector<std::string> ExplicitSwiftModules;
```

Comment:
Can these go into `SearchPathOptions` along with the various search paths? They shouldn't affect the "language" per se.

Reply:
Yeah, I think `SearchPathOptions` probably a more proper home for these paths.